examples/README.md

```
@@ -2,6 +2,16 @@
 
 Here, we collect a number of examples using BlockNote. You can run the examples via the Playground (`npm start` or [online version](https://blocknote-main.vercel.app/)).
 
+Each example directory is a self-contained project — you can open one directly in StackBlitz (`https://stackblitz.com/github/TypeCellOS/BlockNote/tree/main/examples/<group>/<example>`) to try it out or reproduce an issue against the latest release.
+
 ### (contributors) Adding examples
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)